test/parallel/test-util-inspect.js

```diff
@@ -19,11 +19,13 @@
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+// Flags: --expose_internals
 'use strict';
 const common = require('../common');
 const assert = require('assert');
 const util = require('util');
 const vm = require('vm');
+const { previewMapIterator } = require('internal/v8');
```

Comment:
Can we test the `SetIterator` as well?

Reply:
I didn't add a test here, I  just updated an existing test that was using internals.